src/enumMonth/Main.java

```diff
@@ -0,0 +1,29 @@
+package enumMonth;
+
+public class Main {
+    public static void main(String[] args) {
+        Month m = Month.MARCH;
+        System.out.println(String.format("%s has %d days", m.toString(), m.getDaysInAMonth()));
+
+        System.out.println("\nAll months of the Year");
+        for (Month month : Month.values()) {
+            System.out.println(month.toString());
+        }
+
+        System.out.println("\nHolidays in April");
+        Month.APRIL.printHolidays();
+
+        System.out.println();
+        try {
+            Month.holidayCheck("March 21");
+        } catch (Exception e) {
+            System.out.println("Something went Wrong while calculating the date");
+        }
+        try {
+            Month.holidayCheck("March 8");
```

Comment:
your API should throw Exception and with a detailed message what is wrong in the case when you gave incorrect string, and not just indexoutofbound